airflow/providers/elasticsearch/hooks/elasticsearch.py

```diff
@@ -93,3 +95,53 @@ def get_uri(self) -> str:
                 uri += '&'
 
         return uri
+
+
+class ElasticsearchHook(ElasticsearchSQLHook):
+    """
+    This class is deprecated and was renamed to ElasticsearchSQLHook.
+    Please use `airflow.providers.elasticsearch.hooks.elasticsearch.ElasticsearchSQLHook`.
+    """
+
+    def __init__(self, *args, **kwargs):
+        warnings.warn(
+            """This class is deprecated.
+            Please use `airflow.providers.elasticsearch.hooks.elasticsearch.ElasticsearchSQLHook`.""",
+            DeprecationWarning,
+            stacklevel=3,
+        )
+        super().__init__(*args, **kwargs)
+
+
+class ElasticsearchPythonHook(BaseHook):
+    """
+    Interacts with Elasticsearch. This hook uses the official Elasticsearch Python Client.
+
+    :param hosts: list: A list of a single or many Elasticsearch instances. Example: ["http://localhost:9200"]
+    :param es_conn_args: dict: Additional arguments you might need to enter to connect to Elasticsearch.
+                                Example: {"ca_cert":"/path/to/http_ca.crt", "basic_auth": "(user, pass)"}
```

Comment:
It looks like we need also a connection.rst to explain this additional settings for the connection